content/attachments/c0ceaa82-e37c-4b86-a766-59751c6759db/c0ceaa82-e37c-4b86-a766-59751c6759db-příloha.ttl

```diff
@@ -1,13 +1,12 @@
 @prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
 @prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
 
-<https://slovník.gov.cz/datový/pracovní-prostor/pojem/přílohový-kontext/instance342937355/diagram/4d828587-f0ca-44e0-8735-4797ee304146>
-  a <http://onto.fel.cvut.cz/ontologies/application/ontoGrapher/diagram>, <https://slovník.gov.cz/datový/pracovní-prostor/pojem/přílohový-kontext>;
-  <http://onto.fel.cvut.cz/ontologies/application/ontoGrapher/id> "4d828587-f0ca-44e0-8735-4797ee304146";
+<https://slovník.gov.cz/datový/pracovní-prostor/pojem/příloha/instance-c0ceaa82-e37c-4b86-a766-59751c6759db>
+  a <http://onto.fel.cvut.cz/ontologies/application/ontoGrapher/diagram>, <https://slovník.gov.cz/datový/pracovní-prostor/pojem/příloha>;
```

Comment:
The type of the attachment should be the actual type (og:diagram this case). Optionally we could also add pp:příloha, but it is redundant IMO.

Reply:
Sounds great

Reply:
Solved https://github.com/opendata-mvcr/ssp/blob/master/content/attachments/instance-bff068bf-d294-4e8d-bd66-468169d10db2/instance-bff068bf-d294-4e8d-bd66-468169d10db2-p%C5%99%C3%ADloha.ttl